generators/app/prompts.js

```diff
@@ -344,6 +356,17 @@ function askForEntityComponentsOperations(generator, done) {
                 generator.props.autoOffsetResetPolicy = answers.autoOffsetResetPolicy;
             }
 
+            if (answers.sendOrderedMessages) {
+                if (generator.props.currentEntity === constants.NO_ENTITY) {
+                    const uppercasePrefix = utils.transformToJavaClassNameCase(generator.props.currentPrefix);
+                    generator.props.entitiesOrder.push(uppercasePrefix);
+                    generator.props.entitiesOrder[uppercasePrefix] = answers.sendOrderedMessages;
+                } else {
+                    generator.props.entitiesOrder.push(generator.props.currentEntity);
+                    generator.props.entitiesOrder[generator.props.currentEntity] = answers.sendOrderedMessages;
```

Comment:
these entitiesOrder Push could be refactoring into method pushEntitiesOrder(entityName) to avoid code duplication